src/Raven.Studio/typescript/viewmodels/common/detectBrowser.ts

```diff
@@ -0,0 +1,43 @@
+import viewModelBase = require("viewmodels/viewModelBase");
+import studioSettings = require("common/settings/studioSettings");
+
+class detectBrowser extends viewModelBase {
```

Comment:
do we want to extend from view model base?

Reply:
done - removed